cfgov/v1/models/base.py

```diff
@@ -217,6 +220,29 @@ def specific_categories_query(block, parent_slug):
         return {search_type: queryset for search_type, queryset in
                 related.items() if queryset}
 
+    def flush(self):
+        if not settings.ENABLE_AKAMAI_CACHE_PURGE:
+            return False
+        auth, fast_purge_url = get_akamai_credentials()
+        page_relative_path = self.url_path.replace("cfgov/", "")
+        headers = {'content-type': 'application/json'}
+        payload = {
+            'action': 'invalidate',
+            'hostname': settings.WAGTAIL_SITE_NAME,
+            'objects': [
+                page_relative_path
+            ]
+        }
+        resp = requests.post(
+            fast_purge_url,
+            headers=headers,
+            data=json.dumps(payload),
+            auth=auth
+        )
+        if resp.status_code == 201:
+            return True
+        return False
```

Comment:
Yes, meant to add some kind of logging here.  If we were to raise an exception, it would only be displayed to Wagtail users, but I worry this would be confusing, since it'd happen upon publish/unpublish.  While it's good for us to know the flush isn't working, having the page error out on an action like that would make it seem that the page did not successfully publish/unpublish. ... Although, I guess in practice, it didn't if we can't flush the site. So maybe that's not that bad?  Curious what @sarahsimpson09 and Jordana think.